ParisBNF/abb/BNFabb30.xml

```diff
@@ -0,0 +1,720 @@
+<incipit xml:lang="gez"><hi rend="rubric">ወሀሎ᎓ አሐዱ᎓ ብእሲ᎓ ዘይነብ</hi>ር᎓ ባቢሎን᎓
+ወስሙ᎓ ኢ<hi rend="rubric">ዮአቄም᎓ ወአውሰበ᎓ ብእ</hi>ሲት᎓ እንተ᎓ ስማ᎓ ሶስና᎓</incipit>
+  <explicit xml:lang="gez">ፈቀዱ᎓ ይቅትልዎ᎓
+    ወአውርድዎሙ᎓ ውስተ᎓ ግበ᎓ አናብስት᎓ ወበልዕዎሙ᎓ ሶቤሃ᎓ በቅድሜሆሙ፨
+  </explicit>
+  <note>Book of Daniel. The division into parts is
+   indicated by the rubricated word <foreign>ክፍል</foreign> (sometimes
+   abbreviated as <foreign>ክፍ</foreign> or <foreign>ክ</foreign>),
+   followed by numbers 1 to 99 (some numbers are missing).</note>
+</msItem>
+
+      <msItem xml:id="ms_i7">
+    <locus from="164va" to="201rb"/>
+    <title type="complete" ref="LIT1672Isaiah"/>
+    <incipit xml:lang="gez"><hi rend="rubric">ራእየ᎓ ዘርእየ᎓ ኢሳይያስ᎓ ወ</hi>ወልደ᎓
+      አሞጽ᎓ ዘርእየ᎓ በእ<hi rend="rubric">ንተ᎓ ይሁዳ᎓ ወበእንተ᎓ ኢየ</hi>
+    ሩሳሌም፨</incipit>
+    <explicit xml:lang="gez">ወይሬእዩ᎓ አብድንቲሆሙ᎓ ለእለ᎓ ዓለዉኒ᎓ ወዕፄሆሙ᎓ ዘይነውም᎓
+    ወእሳቶሙ᎓ ዘኢይጠፍእ᎓ ወይከውኑ᎓ አርአያ᎓ ለኵሉ᎓ ዘሥጋ፨
+    </explicit>
+    <colophon xml:lang="gez" xml:id="coloph10">
+    <locus target="#201rb"/>
+    ተፈጸመ᎓ ትንቢት᎓ ዘኢሳይያስ᎓ ወልደ᎓ አሞጽ፨ በረከቱ᎓ ትኩን᎓ ምስሌነ᎓ ለዓለመ᎓ ዓለም᎓ አሜን፨
+  </colophon>
+    <note>No division into parts, or it has not been executed
+      (small lacunas throughout
+      the text might have been intended for
+      the rubricated word <foreign>ክፍል</foreign>
+      and number, that were not inserted).</note>
+    </msItem>
+
+    <msItem xml:id="ms_i8">
+     <locus from="201rb" to="203rc"/>
+     <title type="incomplete" ref="LIT5855GIyorgisWaldaAmExcerpt"/>
+  <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡
+    ፩፡ አምላክ፡ በስ</hi>መ᎓ እግዚአብሔር᎓ መሐሪ᎓ ወመተሳህል᎓ ንዌጥን᎓ ጽሒፈ፡ መጽሐፍ፡ ዘይነግር፡
+    ኍልቍ፡ ዓመታተ፡ ዓለም፡ ዘአስተጋብአ፡ አብ፡ ክቡር፡ ወትሩፍ፡ ጊዮርጊስ፡ ወልደ፡ አሚድ፡ ዘብሔረ፡ ግብጽ፡
+  </incipit>
+      <note>The text starts like the Chronicle of Giyorgis
+      Walda ʿAmid, but switches to the chronology of Ethiopian kings.
+      It ends with <persName ref="PRS9449Tewoflos">King Tewoflos</persName>
+      who reigned in <date>1708-11</date>, but the last two lines are scratched out.
+    </note>
+  </msItem>
+
+
+      </msContents>
+
+          <physDesc>
+          <objectDesc form="Codex">
+              <supportDesc>
+              <support>
+             <material key="parchment">Parchment.</material>
+            </support>
+              <extent>
+                <measure unit="leaf">203</measure>
+                  <measure unit="leaf" type="blank"/>
+                    <dimensions type="outer" unit="mm">
+                  <height>240</height>
+                  <width>271</width>
+                    </dimensions>
+                      </extent>
+                      <foliation>
+                          <note>
+                      Foliation of the Bibliothèque nationale de France in the upper
+                        right corner of the recto sides.
+                          </note>
+                      </foliation>
+                      <collation>
+                        <note>Ff. 201-202, with slightly enlarged written
```

Comment:
you coulf mark up with <locus>, both for consistency in the rendering of f./ff./fol. etc. and direct linking to the precise place in the manuscript (which is even already digitised)

Reply:
Ok.